### PR TITLE
Add company to chm api

### PR DESCRIPTION
**Pull request by allisonchihak**

### CODE REVIEW

File: mews-api.md [hunk 1]
```diff
@@ -709,14 +725,31 @@ There are certain rules that need to be followed in order for Mews to process th
...
 | `nationalityCode` | `string` | optional | [ISO 3166-1 alpha-2 country code](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) - two letter country code. |
 | `languageCode`| `string`| optional | Language [code](https://msdn.microsoft.com/en-us/library/ee825488) of the communication language of the customer. This language will be used as the default for communication with the customer. |
 | `address` | [`Address`](mews-api.md#address) object | optional | Represents address. |
 
+#### Company
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+|`id` | `string` | optional | Identifier of a company. |
+|`iata` | `string` | optional | IATA number of a travel agency. |
```
Comment (vitasamek) [inline mews-api.md]: a or the everywhere?

File: mews-api.md [hunk 2]
```diff
@@ -709,14 +725,31 @@ There are certain rules that need to be followed in order for Mews to process th
...
+|`name` | `string` | optional | Name of a company or travel agency. |
+| `contact` | [`Contact`](mews-api.md#contact) object | optional | Company or travel agency contact information. |
+
+#### Contact
+
+| Property | Type |  | Description |
+| --- | --- | --- | --- |
+| `email` | `string` | _recommended_ | Company email address. |
```
Comment (vitasamek) [inline mews-api.md]: this is a general Contact DTO. Should not contain company or travel agency words.

Comment (vitasamek) [reply mews-api.md]: check the state of the reservation array in the creation example.
There is incorrect value 2

File: mews-api.md [hunk 3]
```diff
@@ -709,14 +725,31 @@ There are certain rules that need to be followed in order for Mews to process th
...
+| `telephone` | `string` | optional | Company telephone number. |
+| `address` | [`Address`](mews-api.md#address) object | optional | Company address. |
```
Comment (patrikmiksovsky95) [inline mews-api.md]: All those 'Company' prefixes are in  `### company ` block

**Pull request merged**